src/view/actions/setConsent.jsx

```diff
@@ -61,23 +65,25 @@ const purposesEnum = {
 const getInitialValues = ({ initInfo }) => {
   const {
     instanceName = initInfo.extensionSettings.instances[0].name,
-    consent
+    preferences
   } = initInfo.settings || {};
 
   const initialValues = {
     instanceName
   };
 
-  if (typeof consent === "string") {
+  if (typeof preferences === "string") {
     initialValues.option = purposesEnum.DATA_ELEMENT;
-    initialValues.dataElement = consent;
+    initialValues.dataElement = preferences;
   } else if (
-    consent &&
-    consent.general &&
-    (consent.general === purposesEnum.IN ||
-      consent.general === purposesEnum.OUT)
+    preferences &&
+    preferences.length > 0 &&
```

Comment:
The schema ensures that the rest of these conditions will be true if `preferences` is truthy, so feel free to remove them.